src/main/java/seedu/address/model/util/attributes/Amount.java

```diff
@@ -27,7 +27,7 @@
      * The cost of a Service in Singapore Dollars.
      * @param amount double value input representing dollars and cents.
      */
-    public Amount(double amount) {
+    public Amount(Double amount) {
```

Comment:
I feel that double should remain as a primitive type? If Double means it can accept NULL